pkg/services/alerting/notifier.go

```diff
@@ -126,7 +126,7 @@ func (n *notificationService) uploadImage(context *EvalContext) (err error) {
 	renderOpts := rendering.Opts{
 		Width:           1000,
 		Height:          500,
-		Timeout:         alertTimeout / 2,
+                Timeout:         int(float64(alertTimeout) * 0.9) 
```

Comment:
you need to cast this to `time.Duration`.
And run `gofmt -w -s ./pkg` to get formating.

Reply:
will fix thanks!